frontend/tests/project-creation.spec.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { test, expect } from '@playwright/test';
+
+const nearPixelArea = 50; // Specifies how far away an element can be when using near
+
+test('test the full project creation workflow', async ({ page }) => {
+  const testProjectName = `test-project-name-${Math.trunc(
+    Math.random() * 1000
+  )}`;
+  const testProjectDescription = 'test-project-description';
+
+  await page.goto('http://localhost:4200/');
```

Comment:
The localhost URL should be configurable, it would allow testing against deployed applications (running on a specific URL).